src/js/graphSelector.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2016 Michael Warren Skirpan
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.dataDashboard.graphSelector", {
+        gradeNames: ["fluid.viewComponent"],
+        selectors: {
+            graphSelection: ".floec-graphSelector",
+        },
+        model: {
+            graph: null
+        },
+        events: {
+            graphTypeSelected: null
+
+        },
+        listeners: {
+            "onCreate.setSelectorOptions" : "floe.dataDashboard.graphSelector.setOptions",
+            "onCreate.bindDOMChange" : "floe.dataDashboard.graphSelector.optionsBinding"
+                
+        }
+
+    });
+
+    floe.dataDashboard.graphSelector.optionsBinding = function (that) {
+        var selector = that.locate("graphSelection");
```

Comment:
"selector" is a funny name for these fetched DOM elements. The selector is actually the string ".floec-graphSelector" which appears on the RHS of the selectors block. More idiomatic to simply name them after their selector name = e.g. var graphSelection = that.locate( ..... etc.